python/revng/ptml/actions.py

```diff
@@ -0,0 +1,97 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+from dataclasses import dataclass
+from typing import Callable, Dict, List, Optional, Tuple, Union
+
+
+@dataclass(frozen=True)
+class Location:
```

Comment:
`Location`s are not PTML-specific.
Please review all the new concepts you introduced and push those non-PTML specific out of `/ptml/`.

This confuses people, for instance Fez was thinking that locations are something specific to PTML, but this is not the case.
Also the documentation is problematic in this sense.
Let's not make this situation worse.

Reply:
Moved to `revng/support/location.py`